.expeditor/scripts/generate_verify_pipeline.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
```

Comment:
An "unofficial" pattern is to put scripts like these either in the root of the directory, or in the `.expeditor/buildkite` folder, to allude to their usage.